src/vellum/workflows/nodes/core/inline_subworkflow_node/node.py

```diff
@@ -102,6 +102,11 @@ def run(self) -> Iterator[BaseOutput]:
                 outputs = event.outputs
             elif event.name == "workflow.execution.rejected":
                 raise NodeException.of(event.error)
+            elif event.name == "workflow.execution.paused":
+                raise NodeException(
+                    code=WorkflowErrorCode.INVALID_OUTPUTS,
+                    message="Subworkflow unexpectedly paused",
+                )
```

Comment:
instead of `raise`ing here, define an optional `exception` variable and set it above. This is basically what TryNode, MapNode, ToolCallingNode, etc do